src/components/Dashboard.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div class="h-full w-full flex justify-center items-center bg-[#141e36] flex-col">
+        <h1 class="text-5xl bg-transparent text-white mb-8">Dashboard</h1>
+        <h2 class="text-3xl text-white italic mb-4">Total amount: {{ totalAmount }}</h2>
+        <h3 class="text-2xl text-white italic">Transaktonen</h3>
```

Comment:
Es wäre sinnvoller, text-white global zu definieren, anstatt es in jeder Codezeile zu wiederholen.